datamonster_api/lib/datamonster.py

```diff
@@ -211,35 +211,33 @@ def get_data(
         # todo: support multiple companies
         self._check_param(company=company, datasource=datasource)
 
-        params = {"companyId": company.id}
+        params = {"section_pk": company.id}
```

Comment:
Commented on the other PR about this. Is this an interface change? Why are we doing it? Design doc still uses companyId

Reply:
we just want the endpoint to be simple passthrough to oasis, so the client originates the proper keys and the endpoint just does some validation. i wouldn't call this an interface change, just a change in the parameters we pass.

We will need an interface change (or a newer `get_data_2`) to support estimates since company is no longer relevant, but we make it a required parameter

Reply:
Gotcha. Is there a reason to not just make it an optional parameter now? Nobody is using this endpoint yet, right?

Reply:
👍